js/data/locale/und/CA/numplan.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
There should be no numplan.json for all of the North American Numbering Plan (NANP) countries other than the US. There is code that maps these countries to the US numplan. This includes Canada, Bermuda, Virgin Islands, etc. Also, some countries normalize to France and some to Italy. See https://github.com/iLib-js/iLib/blob/development/js/lib/PhoneLocale.js#L180 for all the mappings.